### PR TITLE
add lemon chat

### PR DESCRIPTION
**Pull request by azc5OQ**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by kokomo123 — CHANGES_REQUESTED** [review]
More useful description is needed, along with capitalizing the first letter of h, j, and r in the language. Then it should be all set to merge.

**Comment by azc5OQ** [comment]
done

**Review by kokomo123 — APPROVED** [review]
Thanks! Now i'm deeming this all set to merge 👍🏻

**Comment by azc5OQ** [comment]
so, is this going to be merged or what is the next step
is the chat not good enough?

**Comment by azc5OQ** [comment]
so, how about it then?

File: README.md [hunk 1]
```diff
@@ -328,6 +328,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [Jami](https://jami.net/) - Free and universal communication platform which preserves the user's privacy and freedoms (formerly GNU Ring). ([Source Code](https://git.jami.net/savoirfairelinux/jami-project)) `GPL-3.0` `C++`
 - [KChat](https://github.com/php-kchat/kchat) - PHP Based Live Chat Application. `Apache-2.0` `PHP`
 - [LeapChat](https://www.leapchat.org/) - Ephemeral, encrypted, in-browser chat rooms. ([Source Code](https://github.com/cryptag/leapchat)) `AGPL-3.0` `Docker/Nodejs/Shell`
+- [LemonChat](https://github.com/azc5OQ/lemon-chat) - Communication system with focus on simplicity and security `GPL-3.0` `Html/Javascript/Rust`
```
Comment (nodiscc) [inline README.md:331]:
```suggestion
- [LemonChat](https://github.com/azc5OQ/lemon-chat) - Communication system with focus on simplicity and security. `GPL-3.0` `Rust`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (azc5OQ) [reply README.md:331]: ok

Comment (azc5OQ) [reply README.md:331]: Im ok with these changes

**Comment by azc5OQ** [comment]
lol

**Comment by azc5OQ** [comment]
I do not know how I can make the simple change that you are asking for. Maybe if I opened new pull request (but I do not know if that is allowed)

Maybe you could just add this small change yourselves? Or is there some other problem?

**Comment by azc5OQ** [comment]
Is this chat not worthy of being "awesome selfhosted?"

**Pull request closed without merging**